widget-loader-acf.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+$key = 'widget_trending_tags';
+$widgetplacement = self::$config['tab_placement'];
+
+$widget_config = array (
+	'key' => $key,
+	'name' => 'trending_tags',
+	'label' => 'Trending Tags',
+	'display' => 'block',
+	'sub_fields' => array (
+		array (
+		'key' => $key . '_basic_details_tab',
+		'label' => 'Basic Details',
+		'type' => 'tab',
+		'placement' => $widgetplacement,
+		),
+		array (
+			'key' => $key . '_tags_manual',
+				'label' => 'Tags',
+				'name' => 'tags',
```

Comment:
The `name` should match the the `key` minus the prefix-y bit (`$key`)

So if your `key` is `widget_trending_tags_tags_manual` then the `name` should be `tags_manual`.